composer.json

```diff
@@ -32,6 +32,7 @@
         "symfony/polyfill-php72": "~1.5"
     },
     "replace": {
+        "symfony/abstractions": "self.version",
```

Comment:
because there are several ones here :)

Reply:
True, but there’s also

- `symfony/cache`
- `symfony/form` 
- etc.

Reply:
I think `contracts` would be better than `abstractions` because final classes are part of API (contract) and they aren't abstract (well, at least not on language level). I hope you have chosen this name because of some serious arguments, not because other well known framework and its similar package ;-)